app/javascript/crayons/Dropdown/__stories__/Dropdown.stories.jsx

```diff
@@ -1,34 +1,34 @@
 import { h } from 'preact';
-import { withKnobs, text } from '@storybook/addon-knobs';
 import './dropdown-css-helper.scss';
 import notes from './dropdowns.md';
-import { Dropdown } from '@crayons';
+import { Dropdown, ButtonNew as Button } from '@crayons';
 
 export default {
   title: 'Components/Dropdowns',
-  decorators: [withKnobs],
-  parameters: { notes },
+  parameters: {
+    docs: {
+      page: notes,
+    },
+  },
 };
 
 export const Default = () => (
   <div className="dropdown-trigger-container">
-    <button
+    <Button
       id="storybook-dropdown-trigger"
-      className="crayons-btn dropdown-trigger"
+      className="dropdown-trigger"
       aria-haspopup="true"
     >
       Click to trigger dropdown
-    </button>
+    </Button>
     <Dropdown
       triggerButtonId="storybook-dropdown-trigger"
       dropdownContentId="storybook-dropdown"
-      className={text('className', 'mb-2')}
     >
       <p>
         Hey, I&apos;m a dropdown content! Lorem ipsum dolor sit amet,
         consectetur adipisicing elit.
       </p>
-      <a href="/">Sequi ea voluptates</a>
```

Comment:
Can we add this back in? The reason the link was there is because all dropdowns should have some interactive content in them, so that we can send focus to it for accessibility

Reply:
👍